src/components/modules/document/common/Upload/modules/Video/modules/UploadSide/UploadSide.vue

```diff
@@ -0,0 +1,220 @@
+<template>
```

Comment:
Can you split this component to small ones? Looks very bloat

Reply:
originally not mine code, just get this from `FrontSideOnly.vue`
and for better we should split `BackSide.vue`, `FrontSideOnly.vue` and this file to new layout